packages/cli/src/index.ts

```diff
@@ -96,13 +89,13 @@ let runChecks = (
 let execLimit = pLimit(4);
 
 async function execCmd(args: string[]) {
-  let { packages } = await getPackages(process.cwd());
+  let { packages, rootDir } = await getPackages(process.cwd());
   let highestExitCode = 0;
   await Promise.all(
     packages.map((pkg) => {
       return execLimit(async () => {
         let { code } = await spawn(args[0], args.slice(1), {
-          cwd: pkg.dir,
+          cwd: path.join(rootDir, pkg.relativeDir),
```

Comment:
isn't this equivalent to this right now?:
```suggestion
          cwd: pkg.dir,
```